content/docs/import/import-from-postgres.md

```diff
@@ -96,7 +96,7 @@ mydumpfile.bak
 
 ## Pipe pg_dump to pg_restore
 
-For small databases (< 1 GB), the standard output of `pg_dump` can be piped directly into a `pg_restore` command to minimize migration downtime:
+For small databases (< 1 GiB), the standard output of `pg_dump` can be piped directly into a `pg_restore` command to minimize migration downtime:
```

Comment:
FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

Reply:
True. Good catches.

Reply:
remove specific sizes